test/functional/wallet_upgradewallet.py

```diff
@@ -345,5 +345,11 @@ def copy_split_hd():
             desc_wallet = self.nodes[0].get_wallet_rpc("desc_upgrade")
             self.test_upgradewallet(desc_wallet, previous_version=169900, expected_version=169900)
 
+        if self.is_bdb_compiled():
```

Comment:
can we also test descriptor+disable_private_keys while we're here?

Reply:
Done